### DISCUSSION TITLE
Compre la licencia pero me da problemas la coapp

### DISCUSSION THREAD

**Discussion by brujillaverde** [aclap-dev/video-downloadhelper#632] [License & Purchase requests]
### What's your issue?

Buenos días
Ya he comentado esto pero me sigue fallando.
Compré la licencia pero no me reconoce la coapp
Me salta todo el rato la pantalla de si permito que la coapp haga cambios en mi equipo y a continuación sale esta :
![image](https://github.com/aclap-dev/video-downloadhelper/assets/169076669/58913338-36f6-46ab-a8f3-08266fd0bc28)
Ya no sé qué hacer!
Gracias de antemano
Está instalado en Windows 11 en Google

### The 10 first digits (only 10) of your license number or email address

sonieta.mate@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
hi,
did you have a look here? https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-not-recognized
